main_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rancher/image-mirror/pkg/config"
+	"github.com/rancher/image-mirror/pkg/regsync"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetRegsyncEntries(t *testing.T) {
```

Comment:
More of a nit/personal preference:
I think it's clearer when we have the input and output of the test near each other, usually in a struct i.e. https://github.com/rancher/rancher/blob/23edf6f5a47f7676cf7ce065c51520ffe23f4b50/pkg/git/git_test.go

Reply:
I changed this to what you suggested in our slack conversation. Please resolve this if you're happy with it.